setup.py

```diff
@@ -579,6 +596,13 @@ def get_common_options(build_ext):
     LIBRARY_DIRS = []
     LIBRARIES = []
 
+    if use_gloo_cpu and use_gloo_cpu != '0':
+        print('Compiling with Gloo')
```

Comment:
What if we always compile with Gloo, how does it affect installation time?

Reply:
One issue to note here is that Mac OS doesn't play nice with Gloo at the moment:

```
In file included from /Users/taddair/Uber/horovod/third_party/gloo/gloo/algorithm.cc:9:
In file included from /Users/taddair/Uber/horovod/third_party/gloo/gloo/algorithm.h:14:
In file included from /Users/taddair/Uber/horovod/third_party/gloo/gloo/math.h:11:
In file included from /Users/taddair/Uber/horovod/third_party/gloo/gloo/types.h:26:
/Users/taddair/Uber/horovod/third_party/gloo/gloo/common/common.h:12:10: fatal error: 'malloc.h' file not found
#include <malloc.h>
         ^~~~~~~~~~
```

Possibly we could check that the machine is Mac and skip Gloo installation automatically (or by default), but enable it for Linux.

Reply:
It's typically 5 seconds longer. With gloo it takes 95, without gloo it takes 90s.

Reply:
FYI: https://github.com/facebookincubator/gloo/issues/182